crmsh/utils.py

```diff
@@ -2287,4 +2253,104 @@ def interface_choice():
     # should consider interface format like "ethx@xxx"
     interface_list = re.findall(r'(?:[0-9]+:) (.*?)(?=: |@.*?: )', out)
     return [nic for nic in interface_list if nic != "lo"]
+
+
+class InterfaceInfo(object):
+    def __init__(self, ipv6=False, second_hb=False):
+        self.ipv6 = ipv6
+        self.nic_info_dict = {}
+        self.second_hb = second_hb
```

Comment:
Maybe better `second_heartbeat`, `hb` is not really meaningfull

Reply:
changed